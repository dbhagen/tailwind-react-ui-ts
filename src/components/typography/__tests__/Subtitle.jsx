import React from 'react'
import { shallow } from 'enzyme'

import { Subtitle } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Subtitle {...props}>Subtitle</Subtitle>)

  return {
    props,
    wrapper,
  }
}

describe('Subtitle', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup({ size: 3 })

    expect(wrapper).toMatchSnapshot()
  })
})
