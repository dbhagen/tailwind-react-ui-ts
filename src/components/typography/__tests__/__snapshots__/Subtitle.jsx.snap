// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle renders matching snapshot 1`] = `
<WithTheme(Title)
  size={3}
  subtitle={true}
>
  Subtitle
</WithTheme(Title)>
`;
